modules/lofreq/1.1/lofreq.smk

```diff
@@ -97,7 +105,8 @@ rule _lofreq_preprocess_normal:
         stdout = CFG["logs"]["lofreq_normal"] + "{seq_type}--{genome_build}/{normal_id}/lofreq_pre.stdout.log",
         stderr = CFG["logs"]["lofreq_normal"] + "{seq_type}--{genome_build}/{normal_id}/lofreq_pre.stderr.log"
     params:
-        opts = CFG["options"]["lofreq"]
+        opts = CFG["options"]["lofreq"],
+        bed = lambda w: _lofreq_get_capspace(w.normal_id, w)
```

Comment:
I added sample_id as a variable to allow the same function to be used for ` _lofreq_run_tumour` and `_lofreq_preprocess_normal`, since the function can't know if the normal or tumour sample is being used